src/grammar.ne

```diff
@@ -1,8 +1,11 @@
 @{%
-  const flattenDeep = require('lodash/flattendeep');
   const appendItem = (a, b) => d => d[a].concat([d[b]]);
   const appendItemChar = (a, b) => d => d[a].concat(d[b]);
 
+  const flattenDeep = arr => {
```

Comment:
why not use https://npmjs.com/array.prototype.flat and `flat(arr, Infinity)`, since it's a standard feature in the language now?

Reply:
I guess I just preferred using the native javascript but anyways replaced with array.prototype.flat now.

Reply:
That package will use native JS in a newer engine, and will have a fallback impl in an older one. Thanks!